pages/common/pprof.md

```diff
@@ -0,0 +1,24 @@
+# pprof
+
+> Comand-line tool for visualization and analysis of profiling data.
+> More information: <https://github.com/google/pprof>.
+
+- Generate a text report from fibbo-profile.pb.gz profile data on fibbo binary:
+
+`pprof -top {{./fibbo}} {{./fibbo-profile.pb.gz}}`
+
+- Generate a graph from fibbo-profile.pb.gz profile data and open it on a web browser:
+
+`pprof -svg {{./fibbo}} {{./fibbo-profile.pb.gz}}`
+
+- Run pprof on interactive mode and customize your report with pprof command:
```

Comment:
I don't understand what does "customize your report with the pprof command" mean ? Can you clarify ?